fuel/data_privacy_policy.md

```diff
@@ -0,0 +1,37 @@
+# Data and Privacy Policy
+
+The Fuel service provides a convenient mechanism to share simulation assets.
+An important aspect involves the collection and distribution of data. This
+privacy policy defines what information is collected and how that
+information is used.
+
+## Information we collect
+
+We are not striving to be an information broker. To that end, we securely
+store the minimal amount of information necessary to create a functional
+service, and we do not share or distribute private information.
+
+
+## Information you give to us. 
```

Comment:
This is technically not a complete sentence and shouldn't be followed by a period
```suggestion
## Information you give to us
```